### PR TITLE
Feature/player 1350

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -217,6 +217,14 @@
   "live": {
     "forceDvrDisabled": false
   },
+  "isVrAnimationEnabled": {
```
Comment (pilievOoyala) [inline skin.json]: How are we using this?  Usually we don't name a variable with a question. That makes me think it's a function.

Comment (KseniiaSokolovaEpam) [reply skin.json]: oh, I meant that values from this object have to be boolean. This values are used for showing animation message for video 360.

**Comment by pilievOoyala** [comment]
LGTM

File: skin.json (hunk 1 above)
Comment (pilievOoyala) [reply skin.json]: It seems ok for now.

**Pull request merged**